service/src/test/groovy/javasabr/mqtt/service/publish/handler/impl/QosMqttPublishInMessageHandlerTest.groovy

```diff
@@ -0,0 +1,13 @@
+package javasabr.mqtt.service.publish.handler.impl
+
+import javasabr.mqtt.service.IntegrationServiceSpecification
+
+abstract class QosMqttPublishInMessageHandlerTest extends IntegrationServiceSpecification {
+  static {
+    //LoggerManager.enable(AbstractMqttPublishInMessageHandler.class, LoggerLevel.DEBUG)
+    //LoggerManager.enable(TrackableMqttPublishInMessageHandler.class, LoggerLevel.DEBUG)
+    //LoggerManager.enable(Qos0MqttPublishInMessageHandler.class, LoggerLevel.DEBUG)
+    //LoggerManager.enable(Qos1MqttPublishInMessageHandler.class, LoggerLevel.DEBUG)
+    //LoggerManager.enable(Qos2MqttPublishInMessageHandler.class, LoggerLevel.DEBUG)
```

Comment:
Are these comments required?

Reply:
@crazyrokr I enable it only during debugging tests